docs/environment.yml

```diff
@@ -1,10 +1,11 @@
 name: qtconsole_docs
 channels:
+  - astrofrog
```

Comment:
Oh cool. It was pulling pyqt from the astrofrog channel. I've gone back to pulling from anaconda in a future commit :smile: